src/sql/workbench/contrib/notebook/find/notebookFindModel.ts

```diff
@@ -565,9 +565,8 @@ export class NotebookFindModel extends Disposable implements INotebookFindModel
 		let searchText: string = input.substr(index);
 		while (findResults.length < maxMatches && searchText.indexOf(exp) > -1) {
 			if (wholeWord) {
-				// word in itself with no spcl characters around \\bword\\b, word ending w/ spcl character \\bword\\s
-				// word begins w/ spcl character \\sword\\b and word that begins and ends with spcl character \\sword\\s
-				let wholeWordRegex = new RegExp(`\\b${this.escapeRegExp(exp)}\\b|\\b${this.escapeRegExp(exp)}\\s|\\s${this.escapeRegExp(exp)}\\b|\\s${this.escapeRegExp(exp)}\\s`);
+				// word in itself with no spcl characters around \\bword\\b, word that begins or ends with spcl character \\sword\\s
```

Comment:
Just spell out special - shortening it like this is just going to confuse people I think